devel/litehtml/Portfile

```diff
@@ -4,11 +4,12 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        litehtml litehtml 0.6 v
+github.setup        litehtml litehtml b5ab35fa12ca27461cc1c0b814ac217410ac8054
+version             0.9-20240402
```

Comment:
This commit is the latest one, which is several months past the release of 0.9. What problems did you encounter with the 0.9 release that prompted you to update further?

Reply:
Now I do not remember. I will try building 0.9.

Reply:
@barracuda156 You did not answer Ryans question here

Reply:
> Now I do not remember. I will try building 0.9.

Reply:
@ryandesign Structure of headers was changed completely, and `litebrowser` did not build with the old version.

P. S. I am not gonna try to fix the old version, it will be a pointless waste of time. If using a newer version is not acceptable, this is your port, just close the PR.